data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/informationtype/InformationTypeController.java

```diff
@@ -71,9 +72,12 @@ public ResponseEntity getInformationTypeById(@PathVariable Long id) {
 			@ApiResponse(code = 404, message = "No InformationTypes found in repository"),
 			@ApiResponse(code = 500, message = "Internal server error")})
 	@GetMapping
-	public Page<InformationTypeResponse> getAllInformationTypes(Pageable pageable) {
+	public RestResponsePage<InformationTypeResponse> getAllInformationTypes(Pageable pageable) {
```

Comment:
Pageable should have default values. If GET is not sending pageable parameters then the default values should be used. This logic will help when data will increase in tables. Right now we do not have much data so it will work fine to show all the records. 

Example values for default pageable: Let's say we can set default elements size 100.

Reply:
I have used the default values of class Pageable, i.e. page=0, size=20. 
Are those values working for you?